tests/test_utils.py

```diff
@@ -0,0 +1,42 @@
+import os
+import numpy as np
+from tempfile import TemporaryDirectory
+from autowebcompat import utils
+from PIL import Image
```

Comment:
Sort the imports in this way:
1) Built-in modules;
2) Modules from third-party libraries;
3) Our modules.